src/parser/rogue/outlaw/modules/Abilities.js

```diff
@@ -65,9 +69,26 @@ class Abilities extends CoreAbilities {
         },
         castEfficiency: {
           suggestion: true,
+          recommendedEfficiency: 0.95,
+          extraSuggestion: 'The only time you should delay casting this is to avoid over-capping Combo Points.',
```

Comment:
Not technically correct, sometimes needs to be saved for burst windows.
This is why I prefer to not be super prescriptive with suggestions, setting ` recommendedEfficiency: 0.95,` already implies it should never be delayed without a good reason.